packages/fluentui/docs/src/components/ComponentDoc/ComponentDoc.tsx

```diff
@@ -11,11 +11,14 @@ import ComponentProps from './ComponentProps';
 import { ComponentDocAccessibility } from './ComponentDocAccessibility';
 import { ThemeContext } from '../../context/ThemeContext';
 import ExampleContext from '../../context/ExampleContext';
-import ComponentPlayground from '../ComponentPlayground/ComponentPlayground';
 import { ComponentInfo } from '../../types';
 import ComponentBestPractices from './ComponentBestPractices';
 import * as _ from 'lodash';
 
+const ComponentPlayground = React.lazy(
+  /* webpackChunkName: "playground" */ () => import('../ComponentPlayground/ComponentPlayground'),
```

Comment:
Does the chunk name comment work correctly with this positioning? I'm used to seeing it look like `import(/* webpackChunkName: "playground" */ '../ComponentPlayground/ComponentPlayground')`

Reply:
Good catch, fixed 👍